GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -70,6 +80,14 @@ public void InitializeDefaultParameterValues()
 
         public void Execute()
         {
+            if (this.StartedByVerb)
```

Comment:
Might be a long shot but are there any tests we can write to validate when `StartedByVerb` is `true` vs when it is `false`?

Reply:
I think we can do that.  Because the functional tests should always start GVFS.Mount using a verb, I can validate that the mount_process log contains `"StartedByVerb":true`